### ISSUE TITLE
store chunked code into DB

### ISSUE THREAD

**Issue by gballet** [ethereum/go-verkle#254]
The code chunks are calculated each time code is executed. This is wasteful, since it need happen only once. Store it inside the database instead.

**Comment by gballet** [comment]
We no longer chunkify the code on each load of the code by the interpreter. This is because chunking is only used when updating the tree, and this is only done upon contract creation. So this optimization is no longer needed.
